{"name": "Test login attempt with wrong credentials", "status": "passed", "steps": [{"name": "test_login_incorrect", "status": "passed", "attachments": [{"name": "screen", "source": "9183b63d-fa67-4a98-93b0-38a467352c18-attachment.png", "type": "image/png"}], "parameters": [{"name": "login_data", "value": "('', '')"}, {"name": "driver", "value": "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"10a226f45b95fd7047479ca5b292907b\")>"}], "start": 1537514882446, "stop": 1537514898258}], "parameters": [{"name": "login_data", "value": "('', '')"}], "start": 1537514882445, "stop": 1537514898258, "uuid": "22b6a881-40ec-4313-97ea-b5dfa07bc7fe", "historyId": "f4349e5ae2ba336cc5543edae9a81784", "fullName": "src.tests.test_selenium.TestLogin#test_login_incorrect[login_data2]", "labels": [{"name": "feature", "value": "User management"}, {"name": "severity", "value": "normal"}, {"name": "tag", "value": "UI"}, {"name": "story", "value": "Login"}, {"name": "parentSuite", "value": "src.tests"}, {"name": "suite", "value": "test_selenium"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "odp1210013"}, {"name": "thread", "value": "10016-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "src.tests.test_selenium"}]}